src/test/scala/lms/transformation/test_distributed_tensor.scala

```diff
@@ -155,5 +155,31 @@ class FixedSizeDistributedTensorTest extends TutorialFunSuite {
   //   System.out.println(indent(driver.code))
   // }
 
+
+  test("conv") {
+    val driver = new CompilerCDistributedTensor[Int, Unit] {
+      import FixedSizeDistributedTensorTypeLess._
+      import scala.collection.immutable.Seq
```

Comment:
need to import immutable.Seq, otherwise has error `required immutable.Seq but found Seq`
Maybe another way to fix that?

Reply:
interesting. Not sure why but likely we `import scala.collection.mutable` somewhere.